packages/sage/src/commands/export.ts

```diff
@@ -114,7 +119,7 @@ async function exportModule(module: string, project: ProjectInfo, commandOptions
 	}
 
 	logger.debug(`Creating plugin project in "${color.bold(exportPath)}"...`)
-	await exec(`${cmd(packageExecutor)} create-robo ${options.join(' ')}`, {
+	await exec(`${cmd(packageExecutor)} create-robo --robo-version ${project.roboversion} ${options.join(' ')}`, {
```

Comment:
Please reuse the `options` array above for this. Append the `--robo-version` to that array the same way we're doing that for the other options. Just to keep things a bit cleaner.~